pkg/k8s/namespaces/namespace.go

```diff
@@ -146,7 +147,7 @@ func (n *Namespaces) DestroyWithLabel(ctx context.Context, ns string, opts Delet
 			return err
 		}
 		gvk := obj.GetObjectKind().GroupVersionKind()
-		if gvk.Kind == volumeKind && !opts.IncludeVolumes {
+		if isStorage(gvk.Kind) && !opts.IncludeVolumes {
 			oktetoLog.Debugf("skipping deletion of pvc '%s' because of volume flag", m.GetName())
```

Comment:
I think we should update the log message to include `ValumeSnapshot` as a possible kind to skip. Something like: `skipping deletion of %s '%s' because of volume flag, gvk.Kind, m.GetName()`

Reply:
+1 to updating the log

Reply:
Done!